src/components/atoms/Skeleton/Line.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// I N T E R F A C E S
+export type LineWidth = 'sm' | 'md' | 'lg'
+export type LineType = 'text' | 'title'
+export interface LineSkeletonProps {
+  width?: LineWidth
+  className?: string
+  style?: React.CSSProperties
+  type?: LineType
+}
+
+// C O M P O N E N T
+export const LineSkeleton: React.FC<LineSkeletonProps> = ({
+  width = 'lg',
+  className = '',
+  style,
+  type = 'text'
+}) => {
+  return (
+    <StyledLineSkeleton style={style} className={`line-skeleton ${className} width-${width} type-${type}`} />
+  )
+}
+
+// S T Y L E S
+const StyledLineSkeleton = styled.span`
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  opacity: .25;
+  background: linear-gradient(to right, var(--color-white) 8%, var(--color-white-70) 18%, var(--color-white) 33%);
+  background-size: 1200px 104px;
+  border-radius: var(--radius);
+  animation-duration: 1s;
+  animation-fill-mode: forwards;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+  animation-name: placeHolderShimmer;
+
+  @keyframes placeHolderShimmer {
+    0%{ background-position: -600px 0; }
+    100%{ background-position: 600px 0; }
+  }
```

Comment:
Would be nice to find a way to declare this animation only once 🤔  I guess there was already such a ticket if my memory is not to bad 😄

Reply:
Hmm yea, we only got a basic animation component yet. This uses a library though, so there are not really any globaly defined animations yet :D But you are right, it would be nice if it was that way